internal/backend/pluggable/pluggable.go

```diff
@@ -105,6 +105,9 @@ func (p *Pluggable) Configure(config cty.Value) tfdiags.Diagnostics {
 	req := providers.ConfigureStateStoreRequest{
 		TypeName: p.typeName,
 		Config:   config,
+		Capabilities: providers.StateStoreClientCapabilities{
+			ChunkSize: DefaultStateStoreChunkSize,
+		},
```

Comment:
I realised that we never sent the default chunk size data from Core to the provider at the start of chunk size negotiation 😬